chainer/functions/lstm.py

```diff
@@ -179,5 +190,21 @@ def lstm(c_prev, x):
     `Long Short-Term Memory in Recurrent Neural Networks \
     <http://www.felixgers.de/papers/phd.pdf>`_.
 
+    .. admonition:: Example
+
+        Assuming ``y`` is current input signal, ``c`` is previous cell state,
+        and ``h`` is previous output signal from ``lstm`` layer with all the
+        length are ``n_units``. Most typical prepareation of ``x`` is:
```

Comment:
-> Assuming ``y`` is **the** current input signal, ``c`` is **the** previous cell state, and ``h`` is **the** previous output signal from **an** ``lstm`` **function**. **Each of ``y``, ``c``, and ``h`` has ``n_units`` channels**. Most typical **preparation** of ``x`` is: